### PR TITLE
Whoops, meant to open a PR for all the commits

### PR DESCRIPTION
**Pull request by benjreinhart**
Fixing Bugs

### CODE REVIEW

File: src/index.coffee [hunk 1]
```diff
@@ -166,7 +168,7 @@ exports.cjsify = (entryPoint, root = path.cwd(), options = {}) ->
         right:
           type: 'CallExpression'
           callee: { type: 'Identifier', name: 'require' }
-          arguments: [{ type: 'Literal', value: relativeResolve root, entryPoint }]
+          arguments: [{ type: 'Literal', value: relativeResolve root, entryPoint, process.cwd() }]
```
Comment (michaelficarra) [inline src/index.coffee]: `root`, not `process.cwd()`. So really, just remove the argument, right?

Comment (benjreinhart) [inline src/index.coffee]: We want `cwd` for this. But I just realized I could just pass `process.cwd()` as the first argument too and leave off the third.

Comment (michaelficarra) [inline src/index.coffee]: Why does it matter where it's being called from? There are 3 locations we have to worry about here:
- the project root, `root`
- the directory containing the file currently being processed, `path.dirname filename`
- the commonjs-everywhere directory with core modules, `path.join __dirname, '..', core`

`process.cwd()` doesn't really come into play other than a default value for `root`

Comment (benjreinhart) [inline src/index.coffee]: Am I missing something? This is the entryPoint file, the one given from the command line, which means that it is always either relative from the `cwd` or an absolute path. So we would never want this to be `root` and if it was a core module you'd still have to give a relative or absolute path to that from the command line. So the use cases you pointed out would never matter, correct?

Comment (michaelficarra) [inline src/index.coffee]: <del>You're definitely missing something. This is in a loop that operates over the `worklist`. Just remove it and everything should work fine.</del>


**edit:** I was thinking of line 143. Still, `process.cwd()` is inappropriate as the last argument here.

Comment (benjreinhart) [inline src/index.coffee]: That's out of the loop. This is just for the entryPoint file.

**Comment by michaelficarra** [comment]
:sunglasses:

**Pull request merged**